README.md

```diff
@@ -16,52 +13,60 @@ Configuration will be loaded in the following order of precedence:
 
 # Installation
 
-To install and use the Consul Telemetry Collector you will need a Consul version of 1.15.3 or greater and to authorize communication with the collector on the Service Mesh.
+Documentation on deploying the Consul Telemetry Collector to a Kubernetes runtime are provided in the HCP documentation:
 
-## Deployment instructions
+- [Deploy the Consul telemetry collector](https://developer.hashicorp.com/hcp/docs/consul/monitor/management-plane/observability/telemetry-collector)
 
-### Docs
+## Configuration
```

Comment:
making a dedicated config section that includes `-help` output to show everything